src/ResourceManager/Resources/Commands.Resources/Properties/Resources.resx

```diff
@@ -243,6 +243,27 @@
   <data name="ResourceProviderNotFound" xml:space="preserve">
     <value>Could not find the resource provider '{0}'.</value>
   </data>
+  <data name="ScopeShouldBeginWithSlash" xml:space="preserve">
+    <value>Scope '{0}' must begin with '/'.</value>
+  </data>
+  <data name="ScopeShouldBeginWithSubscriptionsAndResourceGroups" xml:space="preserve">
+    <value>Scope '{0}' should begin with '/subscriptions/&lt;subid&gt;/resourceGroups'.</value>
```

Comment:
>'/subscriptions/&lt;subid&gt;/resourceGroups [](start = 41, length = 44)

isnt the error message misleading where in at the first error we say
scope should be like "/subscriptions/subid/resourcegroups"
and when they correct that to the above we say
scope should be like  "/subscriptions/subid/resourcegroups/rgname/providers"
and so on